src/app/authentication/authentication.module.ts

```diff
@@ -6,6 +6,7 @@ import { AuthenticationRoutingModule } from './authentication-routing.module';
 import { LoginComponent } from './pages';
 import { FormLoginComponent } from './components/form-login/form-login.component';
 import { AuthenticationService } from './services/authentication.service';
+import { AlertComponent } from 'app/shared/components/alert/alert.component';
```

Comment:
@carryall Could we place `AlertComponent` before `AuthenticationService` so all the components stays together?